lib/learn_to_rank/ranker.rb

```diff
@@ -89,14 +91,8 @@ def fetch_new_scores_from_serving(examples)
       JSON.parse(response.body).fetch("results")
     end
 
-    def tensorflow_serving_ip
-      if ENV["TENSORFLOW_SERVING_IP"].present?
-        ENV["TENSORFLOW_SERVING_IP"]
-      elsif %w(development).include? ENV["RACK_ENV"]
-        "reranker"
-      else
-        "0.0.0.0"
-      end
+    def log_error(error)
```

Comment:
I think you've gained an unused function through a rebase

Reply:
Thanks for spotting that - I've removed it!